rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -390,15 +213,17 @@ def check_airdrops(
         - RemoteError if the remote request fails
     """
     found_data: dict[ChecksumEvmAddress, dict] = defaultdict(lambda: defaultdict(dict))
-    data_dir = database.user_data_dir.parent
+    parent_data_dir = database.user_data_dir.parent
     airdrop_tuples = []
     current_time = ts_now()
-    for protocol_name, airdrop_data in AIRDROPS.items():
+    airdrops, poap_airdrops = fetch_airdrops_metadata(database=database)
+
+    for protocol_name, airdrop_data in airdrops.items():
         if airdrop_data.cutoff_time is not None and current_time > airdrop_data.cutoff_time:
             log.debug(f'Skipping {protocol_name} airdrop since it is not claimable after {airdrop_data.cutoff_time}')  # noqa: E501
             continue
 
-        for row in get_airdrop_data(protocol_name, data_dir):
+        for row in get_airdrop_data(airdrop_data.csv_url, protocol_name, parent_data_dir):
             if len(row) < 2:
                 raise UnableToDecryptRemoteData(
                     f'Airdrop CSV for {protocol_name} contains an invalid row: {row}',
```

Comment:
I just noticed that in the docs the `UnableToDecryptRemoteData` should be a remote error here. also I believe it is hwere the `with pytest.raises(UnableToDecryptRemoteData)` that you added in this PR comes from